src/components/logBookModal.html

```diff
@@ -40,3 +60,17 @@ <h5 class="modal-title">Log Book</h5>
         </div>
     </div>
 </div>
+
+<script type="text/html" id="BooleanSettingTemplate">
+    <td class="p-2">
+        <label class="m-0" data-bind="attr: { for: 'checkbox-' + $data.name }, text: `${$data.displayName}:`">
+            setting name
+        </label>
+    </td>
+    <td class="p-2">
+        <input class="clickable" type="checkbox"
+               data-bind="checked: $data.observableValue(), attr: {name, id: 'checkbox-' + $data.name }"
+               onchange="Settings.setSettingByName(this.name, this.checked)"
+        />
+    </td>
+</script>
```

Comment:
This isn't needed, the templates defined in the setting modal file should be available to use over here too